.github/workflows/pr-checks.yaml

```diff
@@ -128,4 +128,77 @@ jobs:
             kubectl get pods -A
           
         - name: Run Tests
-          run: make test
+          run: make integration-test
+
+  chainsaw-e2e-tests:
```

Comment:
No we can't as we see that if we'll use matrix in GHA, as it will create another separate job for each of the items in the matrix and as we see that the job as a whole is dependent and require all the adapters to be up and running in the same job.